install_files/network_firewall/filter-config-pfSense.xml

```diff
@@ -1 +1,459 @@
+	<rule>
+		<id></id>
+		<tracker>1519947737</tracker>
+		<type>pass</type>
+		<interface>opt1</interface>
+		<ipprotocol>inet</ipprotocol>
+		<tag></tag>
+		<tagged></tagged>
+		<max></max>
+		<max-src-nodes></max-src-nodes>
+		<max-src-conn></max-src-conn>
+		<max-src-states></max-src-states>
+		<statetimeout></statetimeout>
+		<statetype><![CDATA[keep state]]></statetype>
+		<os></os>
+		<protocol>tcp/udp</protocol>
+		<source>
+			<address>app_server</address>
+		</source>
+		<destination>
+			<address>external_dns_servers</address>
+			<port>53</port>
+		</destination>
+		<descr><![CDATA[Allow DNS]]></descr>
+		<created>
+			<time>1519947737</time>
+			<username>admin@10.20.1.2</username>
+		</created>
+		<updated>
+			<time>1519947757</time>
+			<username>admin@10.20.1.2</username>
+		</updated>
+	</rule>
+	<rule>
+		<id></id>
+		<tracker>1519947791</tracker>
+		<type>pass</type>
+		<interface>opt1</interface>
+		<ipprotocol>inet</ipprotocol>
+		<tag></tag>
+		<tagged></tagged>
+		<max></max>
+		<max-src-nodes></max-src-nodes>
+		<max-src-conn></max-src-conn>
+		<max-src-states></max-src-states>
+		<statetimeout></statetimeout>
+		<statetype><![CDATA[keep state]]></statetype>
+		<os></os>
+		<protocol>udp</protocol>
+		<source>
+			<address>app_server</address>
+		</source>
+		<destination>
+			<any></any>
+			<port>123</port>
+		</destination>
+		<descr><![CDATA[Allow NTP]]></descr>
+		<updated>
+			<time>1519947791</time>
+			<username>admin@10.20.1.2</username>
+		</updated>
+		<created>
+			<time>1519947791</time>
+			<username>admin@10.20.1.2</username>
+		</created>
+	</rule>
+	<rule>
+		<id></id>
+		<tracker>1519947935</tracker>
+		<type>block</type>
+		<interface>opt2</interface>
+		<ipprotocol>inet</ipprotocol>
+		<tag></tag>
+		<tagged></tagged>
+		<max></max>
+		<max-src-nodes></max-src-nodes>
+		<max-src-conn></max-src-conn>
+		<max-src-states></max-src-states>
+		<statetimeout></statetimeout>
+		<statetype><![CDATA[keep state]]></statetype>
+		<os></os>
+		<source>
+			<network>opt2</network>
+		</source>
+		<destination>
+			<network>lan</network>
+		</destination>
+		<descr><![CDATA[Block all non-whitelisted traffic from OPT2 and LAN]]></descr>
+		<updated>
+			<time>1519947935</time>
+			<username>admin@10.20.1.2</username>
+		</updated>
+		<created>
+			<time>1519947935</time>
+			<username>admin@10.20.1.2</username>
+		</created>
+	</rule>
+	<rule>
+		<id></id>
+		<tracker>1519947935</tracker>
+		<type>block</type>
+		<interface>opt2</interface>
+		<ipprotocol>inet</ipprotocol>
+		<tag></tag>
+		<tagged></tagged>
+		<max></max>
+		<max-src-nodes></max-src-nodes>
+		<max-src-conn></max-src-conn>
+		<max-src-states></max-src-states>
+		<statetimeout></statetimeout>
+		<statetype><![CDATA[keep state]]></statetype>
+		<os></os>
+		<source>
+			<network>opt2</network>
+		</source>
+		<destination>
+			<network>opt1</network>
+		</destination>
+		<descr><![CDATA[Block all non-whitelisted traffic from OPT2 and OPT2]]></descr>
```

Comment:
I inspected each line in the rules and found a nit: `OPT2 and OPT2` -> `OPT2 and OPT1`, fixed in followup commit 91ffda1